src/var/isWindow.js

```diff
@@ -0,0 +1,8 @@
+define( function() {
+	"use strict";
+
+	return function isWindow( obj ) {
+		return obj !== null && typeof obj !== "undefined" && obj === obj.window;
```

Comment:
@mgol is there anything else that I've missed? 

_Sent from my OnePlus ONEPLUS A3003 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_